calico-cloud_versioned_docs/version-20-1/release-notes/index.mdx

```diff
@@ -5,15 +5,32 @@ title: Release notes
 
 # Calico Cloud release notes
 
-## September DD, 2024 (version 20.0.0)
+## September 10, 2024 (version 20.0.0)
 
 ### New features and enhancements
 
-#### Feature 1
+#### Helm customizations
 
-#### Feature 2
+We've added new options for customizing Helm installations:
 
-### Bug fixes
+* You can now enable or disable the Compliance and Packet Capture features at installation.
+* For many Calico Cloud components, you can specify node selectors, tolerations, and resource requests and limits.
+
+For more information, see [Connect a cluster to Calico Cloud](../get-started/install-cluster.mdx) and [Install Calico Cloud as part of an automated workflow](../get-started/install-automated.mdx)
+
+#### Support for ARM64
+
+This release adds support for clusters running on ARM64 architectures.
+
+#### Support for Kubernetes 1.29
```

Comment:
I think this was a copy/paste error, needs to be 1.30 (also need the same change on the following line too)

Reply:
This is fixed.